pkg/gofr/migration/dgraph.go

```diff
@@ -0,0 +1,164 @@
+package migration
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/dgraph-io/dgo/v210/protos/api"
+
+	"gofr.dev/pkg/gofr/container"
+)
+
+// dgraphDS is the adapter struct that implements migration operations.
+type dgraphDS struct {
+	client DGraph
+}
+
+// dgraphMigrator struct implements the migrator interface.
+type dgraphMigrator struct {
+	DGraph
+	migrator
+}
```

Comment:
I'm unsure I get the comment about the fact the dgraphDS adapter is about the migration

I would have not be surprised to see this

```suggestion
// dgraphDS is the adapter struct that implements migration operations.
type dgraphDS struct {
	client DGraph
}

// dgraphMigrator struct implements the migrator interface.
type dgraphMigrator struct {
	dgraphDS
	migrator
}
```

So here, I'm surprised

Reply:
corrected here too